examples/runExperiments.jl

```diff
@@ -10,150 +10,156 @@ include("getWorkUnit.jl");
 
 function runPaperExperiments(numOf2DRefinements::Int=4, numOf3DRefinements::Int=4)
 
-println("******************** 2D experiments ***************************");
```

Comment:
Similar comment as before, it seems like all that happened here is that you tabbed the lines over. In general, when you send a PR for a specific feature, you don't want to include refactorings like these, otherwise developers have to reason about the relevance of the changes (in fact, I'm only assuming that this is just a refactoring, I'm not 100% sure that none of the logic changed). If you want to send refactorings, you would send that as a separate PR; each PR should try to address one specific feature.